reference/configuration/twig.rst

```diff
@@ -121,13 +121,13 @@ application harder to maintain.
 cache
 ~~~~~
 
-**type**: ``string`` **default**: ``'%kernel.cache_dir%/twig'``
+**type**: ``string`` | ``false`` | ``Twig\Cache\CacheInterface`` **default**: ``'%kernel.cache_dir%/twig'``
```

Comment:
What does `Twig\Cache\CacheInterface` stand for here?

Reply:
An class implementing that interface. I guessed that anyone so advanced (crazy?) to create their own cache implementation would understand it.

Reply:
The point, if I don't miss anything, is that Twig expects an object implementing this interface. Every string will be interpreted as a path. But you cannot pass an object using the semantic configuration.